### PR TITLE
OCT-1758 Cypress: adjust post UQ && OCT-1761 Could not change pages inside the "What is this" UQ onboarding

### PR DESCRIPTION
**Pull request by aziolek**
## Definition of Done

1. [ ] Acceptance criteria are met.
2. [ ] PR is manually tested before the merge by developer(s).
    - [ ] Happy path is manually checked.
3. [ ] PR is manually tested by QA when their assistance is required (1).
    - [ ] Octant Areas & Test Cases are checked for impact and updated if required (2).
4. [ ] Unit tests are added unless there is a reason to omit them.
5. [ ] Automated tests are added when required.
6. [ ] The code is merged.
7. [ ] Tech documentation is added / updated, reviewed and approved (including mandatory approval by a code owner, should such exist for changed files).
    - [ ] BE: Swagger documentation is updated.
8. [ ] When required by QA:
    - [ ] Deployed to the relevant environment.
    - [ ] Passed system tests.

---

(1) Developer(s) in coordination with QA decide whether it's required. For small tickets introducing small changes QA assistance is most probably not required.

(2) [Octant Areas & Test Cases](https://docs.google.com/spreadsheets/d/1cRe6dxuKJV3a4ZskAwWEPvrFkQm6rEfyUCYwLTYw_Cc).

### CODE REVIEW

File: client/src/components/Settings/SettingsUniquenessScoreAddresses/SettingsUniquenessScoreAddresses.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const SettingsUniquenessScoreAddresses: FC<SettingsUniquenessScoreAddressesProps
   const showMoreThanOneAddress = addresses.length > 1;
 
   useEffect(() => {
-    if (isFetchingScore || !ref?.current) {
+    if (isFetchingScore || !ref || !ref.current) {
```
Comment (jmikolajczyk) [inline client/src/components/Settings/SettingsUniquenessScoreAddresses/SettingsUniquenessScoreAddresses.tsx:44]: this is the same logic -> longer version

Comment (jmikolajczyk) [reply client/src/components/Settings/SettingsUniquenessScoreAddresses/SettingsUniquenessScoreAddresses.tsx:44]: My suggestion -> `if (isFetchingScore || !ref?.current?.innerText)`

File: client/src/components/Settings/SettingsUniquenessScoreAddresses/SettingsUniquenessScoreAddresses.tsx [hunk 2]
```diff
@@ -50,7 +50,10 @@ const SettingsUniquenessScoreAddresses: FC<SettingsUniquenessScoreAddressesProps
       {
         duration: 1,
         onUpdate(value) {
-          ref.current!.textContent = value.toFixed(0);
+          if (!ref.current) {
+            return;
+          }
+          ref.current.textContent = value.toFixed(0);
```
Comment (jmikolajczyk) [inline client/src/components/Settings/SettingsUniquenessScoreAddresses/SettingsUniquenessScoreAddresses.tsx:56]: `ref.current.innerText = value.toFixed(0);`

**Review by jmikolajczyk — CHANGES_REQUESTED** [review]

File: client/src/components/Settings/SettingsUniquenessScoreAddresses/SettingsUniquenessScoreAddresses.tsx (hunk 1 above)
Comment (aziolek) [reply client/src/components/Settings/SettingsUniquenessScoreAddresses/SettingsUniquenessScoreAddresses.tsx]: Done.

**Review by jmikolajczyk — APPROVED** [review]

**Pull request merged**